docs/score_releases/index.rst

```diff
@@ -0,0 +1,106 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2024 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+SCORE Releases Overview
+=======================
+
+Roadmap
+--------
+Roadmap is a living document.
+
+.. image:: _assets/score_timeline.svg
+   :width: 700
+   :alt: Infrastructure overview
+   :align: center
+
+Overview
+--------
+
+.. image:: _assets/score_mvp.svg
+   :width: 900
+   :alt: Infrastructure overview
+   :align: center
+
+SW Module List
+--------------
+
+.. list-table:: SCORE software modules overview
+   :header-rows: 1
+
+   * - SW Module Name
+     - Committer List
+     - Release
+     - Date
+     - Contribution Request
+     - Repo
+   * - Logging
+     - _
+     - MVP
+     - Q4
+     - _
+     - `mw_log <https://github.com/eclipse-score/inc_mw_log>`_
```

Comment:
To be fully transparent I would also show the link with the `inc_` prefix. After Q1 we can then add the contribution request link and the actual repo.
```suggestion
     - `inc_mw_log <https://github.com/eclipse-score/inc_mw_log>`_
```